rktlet/runtime/helpers.go

```diff
@@ -70,7 +71,11 @@ func parseContainerID(containerID string) (uuid, appName string, err error) {
 	if len(values) != 2 {
 		return "", "", fmt.Errorf("invalid container ID %q", containerID)
 	}
-	return values[0], values[1], nil
+	appName, err = actypes.SanitizeACName(values[1])
```

Comment:
could we have a file listing the limitations (collision between "foobar" and "FooBar")? e.g. `docs/limitations.md`